src/Skoruba.IdentityServer4.Admin.BusinessLogic/Helpers/EnumerableExtensions.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Skoruba.IdentityServer4.Admin.BusinessLogic.Helpers
+{
+    public static class EnumerableExtensions
+    {
+        public static IEnumerable<T> WhereIf<T>(this IEnumerable<T> query, bool condition, Func<T, bool> predicate)
```

Comment:
Similarly here you could have `Func === null` be a replacement for `condition` and default it it `null`